packages/clickable/components/clickable.tsx

```diff
@@ -23,6 +23,10 @@ export interface ClickableProps {
    * Whether or not to reset the :focus outline style
    */
   disableFocusOutline?: boolean;
+  /**
+   * human-readable selector used for writing tests
+   */
+  dataCy?: string;
```

Comment:
This element doesn't have a data-cy.

Reply:
It's not supposed to. `Clickable` just renders it's children, so if you're using it on it's own, you just have to pass the `dataCy` prop